### DISCUSSION TITLE
Analytics?

### DISCUSSION THREAD

**Discussion by angelogulina** [opendatasicilia/informa-covid19#37] [Generale]
Vogliamo degli Analytics di qualche genere? (Google, ecc)

Potrebbe essere interessante per capire da dove vengono le persone che visitano il progetto e dove vanno (se cliccano sulle dashboard o su altro)?

_Just wondering_

**Comment by dennisangemi** [comment]
Secondo me è una buona idea inserirle. Nelle mie dashboard di data studio sono già presenti ;)

**Reply by angelogulina** [reply]
Ah, bene. Dovremmo aggiungere allora un parametro al link su Informa Covid così che tu possa individuare chi arriva alle tue Dashboard da qui. Che ne pensi?

**Reply by dennisangemi** [reply]
Assolutamente d'accordo
